layer/config/__init__.py

```diff
@@ -14,18 +14,3 @@
 )
 from .config_client import ConfigClient  # noqa
 from .config_manager import ConfigManager  # noqa
-
-
-def is_executables_feature_active() -> bool:
-    import os
-
-    if "LAYER_EXECUTABLES" in os.environ:
-        return True
-
-    try:
-        config = ConfigManager().load()
-    except:  # noqa
-        return False
-
-    # enable by default for non production environments
-    return config.url != DEFAULT_URL
```

Comment:
Can we just flip the default rather than remove completely? If a user hits issues, we can tell them to disable it as opposed to have to ship a new version in a rush.

Reply:
I think that @volkangurel was eager to start deleting code.
I'd feel a bit safer flipping the flag as @dpp23 suggested. Here's that option, if we want to go for it: https://github.com/layerai/sdk/pull/367

Reply:
I was, but I agree with @dpp23 that we can do that a bit later.